### PR TITLE
[AAE-11400] Error message in stack trace list

### PR DESCRIPTION
**Pull request by wojciech-piotrowiak**

### CODE REVIEW

File: activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/IntegrationErrorReceivedEventHandler.java [hunk 1]
```diff
@@ -58,6 +62,28 @@ public void handle(CloudRuntimeEvent<?, ?> event) {
         });
     }
 
+    @NotNull
+    private static List<StackTraceElement> addFullErrorMessageAsFirstStackTraceElement(
```
Comment (graric) [inline activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/IntegrationErrorReceivedEventHandler.java]: Needs to be static?

File: activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/IntegrationErrorReceivedEventHandler.java [hunk 2]
```diff
@@ -58,6 +62,28 @@ public void handle(CloudRuntimeEvent<?, ?> event) {
...
+        CloudIntegrationErrorReceivedEvent integrationEvent) {
+        StackTraceElement newTopStackTraceElement = getNewTopStackTraceElement(integrationEvent);
+        List<StackTraceElement> stackTraceElements = new java.util.ArrayList<>(List.of(newTopStackTraceElement));
+        stackTraceElements.addAll(integrationEvent.getStackTraceElements());
+        return stackTraceElements;
+    }
+
+    @NotNull
+    private static StackTraceElement getNewTopStackTraceElement(CloudIntegrationErrorReceivedEvent integrationEvent) {
```
Comment (graric) [inline activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/IntegrationErrorReceivedEventHandler.java]: Needs to be static?

File: activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/IntegrationErrorReceivedEventHandler.java [hunk 3]
```diff
@@ -58,6 +62,28 @@ public void handle(CloudRuntimeEvent<?, ?> event) {
         });
     }
 
+    @NotNull
+    private static List<StackTraceElement> addFullErrorMessageAsFirstStackTraceElement(
+        CloudIntegrationErrorReceivedEvent integrationEvent) {
+        StackTraceElement newTopStackTraceElement = getNewTopStackTraceElement(integrationEvent);
+        List<StackTraceElement> stackTraceElements = new java.util.ArrayList<>(List.of(newTopStackTraceElement));
```
Comment (graric) [inline activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/IntegrationErrorReceivedEventHandler.java]: Why not to import the package for ArrayList?

File: activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/IntegrationErrorReceivedEventHandler.java (hunk 1 above)
Comment (wojciech-piotrowiak) [reply activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/IntegrationErrorReceivedEventHandler.java]: no, probably generated by IDE

File: activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/IntegrationErrorReceivedEventHandler.java (hunk 3 above)
Comment (wojciech-piotrowiak) [reply activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/IntegrationErrorReceivedEventHandler.java]: fixed

File: activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/IntegrationErrorReceivedEventHandler.java (hunk 2 above)
Comment (wojciech-piotrowiak) [reply activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/IntegrationErrorReceivedEventHandler.java]: no, probably generated by IDE

**Review by graric — APPROVED** [review]

**Pull request merged**